trace_log/resources/spring/camel-context.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does trace_log need a camel-context? Wouldn't it just get incorporated into the phinms and foodNet contexts?